PROJECT

```diff
@@ -8,7 +8,7 @@ layout:
 plugins:
   manifests.sdk.operatorframework.io/v2: {}
   scorecard.sdk.operatorframework.io/v2: {}
-projectName: sailoperator
+projectName: servicemeshoperator3
 repo: github.com/istio-ecosystem/sail-operator
```

Comment:
repo URL should change also?

Reply:
I think it's fine to leave as-is for now